testing/src/com/google/idea/testing/ServiceHelper.java

```diff
@@ -141,8 +142,6 @@
     ServiceContainerUtil.registerServiceInstance(componentManager, key, implementation);
     Disposer.register(
         parentDisposable,
-        () ->
-            ((MutablePicoContainer) componentManager.getPicoContainer())
-                .unregisterComponent(key.getName()));
+        () -> new ComponentManagerWrapper(componentManager).unregisterComponent(key.getName()));
```

Comment:
Even though this can be implemented as a Wrapper, it's not directly necessary. It would also be possible to introduce a static method to `BaseSdkCompat` which takes `componentManager` and `key.getName()` as method parameters. Going for a method in `BaseSdkCompat` is more light-weight and hence preferred. Can you please switch to that approach?

Reply:
Sure, will do. Just double-checking: I'm assuming this should go in `BaseSdkTestCompat` then. Is that correct?

Reply:
Right. Since it's only used from test code, it should go into `BaseSdkTestCompat`. Thanks for double-checking.